### DISCUSSION TITLE
Copy Emote To Cursor Position On Right Click

### DISCUSSION THREAD

**Discussion by Philipp-Codes** [Chatterino/chatterino2#5375] [Ideas]
### Proposal

* The ability to right-click an emote in the current chat to instantly paste it at the text cursor position in the "Send message" input field.
* The addition of a setting in the user preferences to enable / disable the feature.

### Motivation

Having the ability to quickly copy emotes from past messages and include them in a new message saves time and effort, especially when the topics are quickly changing. This feature enables users to promptly convey their thoughts using appropriate emotes relevant to the ongoing conversation, without the need for manual typing or searching. This also comes in handy for long emote names or complicated prefixes. 

### Background

The idea for this feature comes from using the browser extension _Twitch Enhancer_ which already offers this functionality and greatly enhanced my personal experience in regular Twitch chat. 
I constantly notice myself missing this feature in Chatterino and would therefor greatly appreciate its addition to the chat client.

**Comment by pajlada** [comment]
We currently allow for running commands on messages you right clicked - it could be expanded with a replacement for the emote or word you right clicked on. Will convert this into an official feature request later on unless anyone thinks this sounds crazy

**Reply by Mm2PL** [reply]
I would really like for commands to be able to access the element which was clicked. This could be especially powerful in combination with plugins.
